scripts/benchmark.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+LAKE=lake
+BENCH="$LAKE env lean -Dweak.benchmark.runs=5"
+OUT="data/benchmarks"
```

Comment:
Let's add an option to clean all these generated files under `clean_all` in the Makefile. Same for files generated by `profile.sh`.

Reply:
I was a bit wary at first: I tend to run `make clean` when something breaks my build and would not want this to also delete all of my benchmarking data. I guess `make_all` is already a separate target, so it should be fine though

Reply:
@alexkeizer Indeed (but I think you meant `clean_all`, not `make_all`). Mind making that change, and then we'll hit merge.

Reply:
Done